src/documentation/pages/components/breadcrumb/examples/_alternate-separator.vue

```diff
@@ -0,0 +1,60 @@
+<template lang="pug">
+  <ComponentExample title="Alternate Separator" id="alternate-separator" :tabs="exampleTabs">
```

Comment:
You can use titleSize h4 for this title as this title is smaller in size when compared to Additional sizes.

Reply:
For reference, screenshot from production.

<img width="547" alt="Annotation 2022-02-25 152007" src="https://user-images.githubusercontent.com/92325462/155810919-00e78b3a-e163-4b2b-9c13-fa4982e919bf.png">

Reply:
I saw that but it didn't seem to make sense to me, that additional sizes is h2 like examples when it is an example itself, that's why i used h3 for it too 🤔

Reply:
Yea, i did observe the title sizes are inconsistent when compared with production ones and they are looking big if we use h2.